Examples/Python/src/TrackFinding.cpp

```diff
@@ -98,7 +98,6 @@ void addTrackFinding(Context& ctx) {
     ACTS_PYTHON_MEMBER(impactMax);
     ACTS_PYTHON_MEMBER(deltaZMax);
     ACTS_PYTHON_MEMBER(zBinEdges);
-    ACTS_PYTHON_MEMBER(skipPreviousTopSP);
```

Comment:
Since the sorting of SPs is done by default now, we don't need `skipPreviousTopSP` anymore, unless for SSS where we lose the sorting after some coordinates corrections that uses SSS information. From @CarloVarni comments I decided that it was better to "merge" `skipPreviousTopSP` and `useDetailedDoubleMeasurementInfo` which are related and only used by SSS seeding